test/gtest/common/test_obj_size.cc

```diff
@@ -53,7 +53,7 @@ UCS_TEST_F(test_obj_size, size) {
     EXPECTED_SIZE(uct_base_ep_t, 8);
     EXPECTED_SIZE(uct_rkey_bundle_t, 24);
     EXPECTED_SIZE(uct_self_ep_t, 8);
-    EXPECTED_SIZE(uct_tcp_ep_t, 72);
+    EXPECTED_SIZE(uct_tcp_ep_t, 128);
```

Comment:
is is the growер so big? maybe it is possible to rearrange fields somehow?

Reply:
ok, will try

Reply:
rearranging didn't help

1, can declare `ucs_sock_addr_t peer_addr` as `ucs_sock_addr_t *peer_addr`, it will save 8 bytes. It is used only during initialization.
2. can use `uint32_t` for the `length` and `offset` fields. it will save 8 bytes per eac context (i.e. 16 bytes in total)

WDYT?

Reply:
we will have to have ucs_sock_addr_t per ep anyway, right? no point to keep a pointer to it
let's analyze how we can reduce it when all your PRs are ready

Reply:
> we will have to have ucs_sock_addr_t per ep anyway, right?

yep, right

> let's analyze how we can reduce it when all your PRs are ready

ok